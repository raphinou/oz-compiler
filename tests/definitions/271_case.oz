% Pattern conjunctions open records and escaped variables in function arguments
local
   P C=c
in
   proc {P V=fRecord(lab [ f1(a B !C ...) f2 F3 ] M=more(m1 M2 ...) ... )}
      {Show V}
      {Show B}
      {Show F3}
      {Show M}
      {Show M2}
   end
   {P fRecord(lab [f1(a b c d e f) f2 f3] more(m1 m2 m3) rest)}
end
