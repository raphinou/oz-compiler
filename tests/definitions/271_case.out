fRecord(lab [f1(a b c d e f) f2 f3] more(m1 m2 m3) rest)
b
f3
more(m1 m2 m3)
m2
